dss/time_series.go

```diff
@@ -66,7 +67,7 @@ func ReadTimeSeries(filePath string, record string, outputJSON string) ([]byte,
 	cTimeLength := C.int(10) // 10 Characters for a time string
 
 	C.zopen(&ifltab, cFilePath)
-	defer C.zclose(&ifltab)
+	// defer C.zclose(&ifltab) --> Evoking this (either via defer or at the end) exposes a bug.
```

Comment:
It seemed like it was an issue with evoking it within the scope of the function responsible for reading. We never tested doing zopen externally to the scope of the ReadTimeSeries and zclose after the method returns. I wonder if the memory would get corrupted in that scenario as well. This could be a go memory managment issue or a dss zclose issue.